src/SFA.DAS.Tools.Support.Web/Startup.cs

```diff
@@ -60,7 +60,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 var policy = new AuthorizationPolicyBuilder()
                     .RequireAuthenticatedUser()
-                    .RequireRole(_configuration["RequiredRole"])
+                    //.RequireRole(_configuration["RequiredRole"])
```

Comment:
I hope you can remember to add this back in when the staff IDAMS role has been added :smile:

Reply:
done